Dedicated Hosting_Table.md

```diff
@@ -0,0 +1,72 @@
+# Dedicated Hosting_Table
```

Comment:
You can remove this file from the root directory, (aka. this is on the top level folder). Don't delete the one in the content folder.

Reply:
Deleted the file